src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionProviderOptionsHelper.cs

```diff
@@ -119,16 +119,36 @@ internal static ConnectionProviderOptions BuildConnectionProviderOptions()
                     new ConnectionOption
                     {
                         Name = "columnEncryptionSetting",
-                        DisplayName = "Column encryption setting",
-                        Description = "Default column encryption setting for all the commands on the connection",
+                        DisplayName = "Always Encrypted",
+                        Description = "Enables or disables Always Encrypted for the connection",
```

Comment:
Where is this new description coming from? No issues with the change - just not sure I understand the context for the change here.

Reply:
Our Program Manager wanted it to match the verbiage that exists in SSMS for user experience parity.